### PR TITLE
Add current place to nearby place list and filter/sort nearby places

### PR DESCRIPTION
**Pull request by shifucun**
image

### CODE REVIEW

**Comment by shifucun** [comment]
> lgtm, but could you merge in after #342? i prefer the fix there for getting population, which just uses the latest value, not necessarily 2018 (which doesn't always exist for non-US geos)

sg, I will wait for that, also looking at that now.

**Comment by shifucun** [comment]
Now using the latest population

File: server/routes/api/place.py [hunk 1]
```diff
@@ -137,7 +140,7 @@ def child(dcid):
     for place_type in child_places:
         child_places[place_type].sort(key=lambda x: x['pop'], reverse=True)
         child_places[place_type] = child_places[place_type][:CHILD_PLACE_LIMIT]
-    return json.dumps(child_places)
+    return Response(json.dumps(child_places), 200, mimetype='application/json')
```
Comment (beets) [inline server/routes/api/place.py:143]: It doesn't work for one of these handlers which is being called by another function. Please check that all the handlers still return correctly in the chrome network inspector.

Comment (shifucun) [reply server/routes/api/place.py:143]: good to know. Checked a few pages and urls, these changes are okay.

**Pull request merged**